src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,269 @@
+        outlier_melt_df: Outliers determined in EvidenceQC for MELT.
+        outlier_wham_df: Outliers determined in EvidenceQC for Wham.
+        outlier_type: high or low. Determined in EvidenceQC for each of the three callers.
+    Returns:
+        The total number of times that a sample appears as an outlier
+        across all three algorithms for each sample.
+    """
+    # Manta:
+    col_name = get_col_name("manta", outlier_type)
+    dict_manta = dict(zip(outlier_manta_df[ID_COL], outlier_manta_df[col_name]))
+
+    # Melt:
+    col_name = get_col_name("melt", outlier_type)
+    dict_melt = dict(zip(outlier_melt_df[ID_COL], outlier_melt_df[col_name]))
+
+    # Wham:
+    col_name = get_col_name("wham", outlier_type)
+    dict_wham = dict(zip(outlier_wham_df[ID_COL], outlier_wham_df[col_name]))
+
+    # merging all the dictionaries
+    outlier_dicts = [dict_manta, dict_melt, dict_wham]
+    merged_dicts = Counter()
+    for counted in outlier_dicts:
+        merged_dicts.update(counted)
+    all_outliers = dict(merged_dicts)
+    all_outliers_df = pd.DataFrame.from_dict(all_outliers, orient="index").reset_index()
+    all_outliers_df.columns = [ID_COL, outlier_type + "_overall_outliers"]
+    return all_outliers_df
+
+
+def merge_evidence_qc_table(
+        filename_estimated_cn: str,
+        filename_mediancov: str,
+        filename_wgd: str,
+        filename_cnv_qvalues: str,
+        filename_high_manta: str,
+        filename_high_melt: str,
+        filename_high_wham: str,
+        filename_low_manta: str,
+        filename_low_melt: str,
+        filename_low_wham: str,
+        filename_melt_insert_size: str,
+        output_prefix: str) -> None:
+    """
+    Reads the provided TSV files (tab-delimited) and merges all the information in one table
+    serialized to the given output filename.
+    """
+    df_ploidy = read_ploidy(filename_estimated_cn)
+    df_bincov_median = read_bincov_median(filename_mediancov)
+    df_wgd_scores = read_wgd_scores(filename_wgd)
+    df_non_diploid = read_non_diploid(filename_cnv_qvalues)
+    df_manta_high_outlier = read_outlier(filename_high_manta, get_col_name("manta", "high"))
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"))
+    df_total_high_outliers = read_all_outlier(df_manta_high_outlier, df_melt_high_outlier, df_wham_high_outlier, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"))
+    df_total_low_outliers = read_all_outlier(df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, "low")
+    df_melt_insert_size = read_melt_insert_size(filename_melt_insert_size)
```

Comment:
Not necessary to merge / question of preference. When I tested without providing one of the outlier files, I noticed that it produced a column of NA values. I think I might prefer to omit the column if the corresponding file is not provided, but would be open to other views.